src/data.ts

```diff
@@ -0,0 +1,31 @@
+import { LocData } from './types.d.ts';
+
+/** fetchWrapper returns an instance of fetch wrapped for use with a single resource with def data and error handling */
+const fetchWrapper = (url: string, dataHandler?: any, errorHandler?: any) => {
+  return async () => {
+    try {
+      const res = await fetch(url);
+      const data = await res.json();
+      return dataHandler ? dataHandler(data) : data;
+    } catch (err) {
+      return errorHandler ? errorHandler(err) : console.log(err);
+    }
+  };
+};
+
+/** trimLocation returns only the needed data fields from larger location data object*/
+const trimLocation = (
+  data: { city: string; region: string; loc: string },
+): LocData => {
+  return {
+    city: data.city,
+    region: data.region,
+    cordinates: data.loc,
+  };
+};
```

Comment:
There's not much to test here, so I think it would be fine to either skip testing this function, or write a single test that verifies it works as expected. With TypeScript, I don't think you'd even be able to pass in bad data to try to break it.